images/source/cbs-oranges7-test64-common-el7-build.repo

```diff
@@ -0,0 +1,8 @@
+[oranges7-test64-common-el7-build]
+name = CBS Oranges7
+baseurl = http://cbs.centos.org/kojifiles/repos/oranges7-test64-common-el7-build/latest/$basearch
+enabled = 1
+gpgcheck = 0
+sslverify = 0
+skip_if_unavailable = 1
+includepkgs = golang* protobuf*
```

Comment:
This repo has builds for ppc64le and aarch64, it is currently a stopgap. These packages will be moved into a repo owned by the CentOS PaaS SIG once we have ppc64le enabled.

In this case, it is providing golang-1.8.3, but only for aarch64 and ppc64le